### DISCUSSION TITLE
Support custom user-friendly URLs (slug) for Listed

### DISCUSSION THREAD

**Discussion by dmitriy-bodroff** [standardnotes/forum#2707] [Suggestions]
I want to set custom user-friendly URLs (slug) for my posts in Listed because Listed can't convert non-latin characters.

Example URL: https://listed.to/@nablagodel/17336/.

Now, there is no slug.

![image](https://user-images.githubusercontent.com/523055/90028860-db588880-dcc2-11ea-945e-12cb9f90ccd4.png)

I suggest to add new per-post option (https://docs.standardnotes.org/listed/options) for example `slug` (https://en.wikipedia.org/wiki/Clean_URL).

```
---
slug: svyazka-instrumentov-dlya-ukroscheniya-informacionnogo-shtorma
---
```

And, as result, we get follow URL: https://listed.to/@nablagodel/17336/svyazka-instrumentov-dlya-ukroscheniya-informacionnogo-shtorma/

**Comment by JaspalSuri** [comment]
Thanks for the feedback! I'll pass this along to the team for review.

**Comment by atmoio** [comment]
You can already do that. Just click your last URL. You can put anything you want after the ID of the post (https://listed.to/@nablagodel/17336/anything-you-want)

**Comment by dmitriy-bodroff** [comment]
@mobitar, yes, I know it. But I am talking about links in the blog. Its don't include slug.
